Mathlib/Combinatorics/Additive/Behrend.lean

```diff
@@ -172,8 +172,8 @@ nonrec theorem addSalemSpencer_sphere : AddSalemSpencer (sphere n d k : Set (Fin
     { toFun := fun f => ((↑) : ℕ → ℝ) ∘ f
       map_zero' := funext fun _ => cast_zero
       map_add' := fun _ _ => funext fun _ => cast_add _ _ }
-  refine' AddSalemSpencer.of_image (f.toAddFreimanHom (sphere n d k : Set (Fin n → ℕ)) 2) _ _
-  · exact cast_injective.comp_left.injOn _
+  refine (AddMonoidHomClass.isAddFreimanHom f (Set.mapsTo_image _ _)).addSalemSpencer
```

Comment:
The SalemSpencer bits of this file need to be fixed now given recent changes I think.